docs/quick-release-notes/main.go

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2020  Matthew "strager" Glazar
+// See end of file for extended copyright information.
+
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"regexp"
+)
+
+type Tag struct {
+	Name       string `json:"name"`
+	ZipballURL string `json:"zipball_url"`
+	TarballURL string `json:"tarball_url"`
+	Commit     struct {
+		SHA string `json:"sha"`
+		URL string `json:"url"`
+	} `json:"commit"`
+	NodeID string `json:"node_id"`
+}
+
+func getTagsFromAPI(owner string, repo string) []Tag {
+	// https://docs.github.com/en/rest/reference/repos#list-repository-tags
+	pathToTags := fmt.Sprintf("https://api.github.com/repos/%v/%v/tags", owner, repo)
+	resp, err := http.Get(pathToTags)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	sb := string(body)
+	var tagsForEachRelease []Tag
+	json.Unmarshal([]byte(sb), &tagsForEachRelease)
```

Comment:
Nit: Don't convert from `[]byte` to `string` to `[]byte`:
```suggestion
	var tagsForEachRelease []Tag
	json.Unmarshal(body, &tagsForEachRelease)
```